pkg/util/mount/mount_windows_test.go

```diff
@@ -601,3 +602,75 @@ func TestGetFileType(t *testing.T) {
 		}
 	}
 }
+
+func TestFormatAndMount(t *testing.T) {
+	fakeMounter := ErrorMounter{&FakeMounter{}, 0, nil}
+	execCallback := func(cmd string, args ...string) ([]byte, error) {
+		for j := range args {
+			if strings.Contains(args[j], "Get-Disk -Number") {
```

Comment:
Is there anyway here that you can validate the passed in arguments to the command are expected?  For example, validate that fstype is set correctly.  And if invalid device is set, then have the command return error?

Reply:
@msau42 I leave the fstype check in the actual powershell command, that is if fstype is set incorrect, that powershell command will return error, there is nothing to do in the fake `execCallback`; and for the invalid device, it already return error in the test case. I have squashed all commits, could you take a final review, thanks.